src/tools/set_videomode

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/perl
+
+use strict;
+
+my $usage = <<XXX;
+set_videomode [MODE][,DPI]
+
+Sets video mode MODE using xrandr. If DPI is specified, also adjusts physical
+screen size to match the dpi value. If MODE is missing, only DPI is adjusted.
+
+If the mode can't be set, the old video mode is kept.
+
+To be precise, it changes the first connected display.
+XXX
+
+my ($mode, $dpi) = split /,/, @ARGV[0];
+$dpi += 0;
+
+die $usage unless $mode ne "" || $dpi;
+
+open my $log, ">>/var/log/YaST2/y2videomode.log";
+
+my $xrandr = "xrandr";
+
+my @randr = `$xrandr 2>&1`;
```

Comment:
Is it a good idea to redirect stderr to stdout here? What kinds of errors might xrandr throw? Maybe they are important?